backend/src/modules/tournaments-registrations/core/domain/TournamentRegistrations.ts

```diff
@@ -98,9 +98,12 @@ export function closeTournamentRegistrations(
     throw new Error('Only opened registrations can be closed!');
   }
   const registeredPlayersCount = state.registeredPlayers.length;
-  if (state.registeredPlayers.length < MIN_TOURNAMENT_PLAYERS) {
+  if (registeredPlayersCount < MIN_TOURNAMENT_PLAYERS) {
```

Comment:
Jak to się stało , że linijka 100 jest dodana wcześniej a wykorzystane  nie było?
Błąd githuba?

Reply:
była - w opisie errora :-)